prime-router/docs/proposals/0013-how-to-build-a-fhir-engine.md

```diff
@@ -0,0 +1,77 @@
+# How to build a FHIR engine
+Directions from Here to There.
+
+## Working Agreement
+-   For any new feature, we will not add any hidden magic to existing behaviors, rather we will create new, properly encapsulated concerns
+-   For every new feature implemented, we will evaluate how we can implement functionality that aligns to this future state architecture
+-   In order to realize a revised architecture that enables longevity and improved velocity of the engineering team, we acknowledge that new features may take longer to implement
+
```

Comment:
'There will be clearly defined milestones that allows for incremental, testable deployments.